lib/util/player_configuration.js

```diff
@@ -332,6 +332,8 @@ shaka.util.PlayerConfiguration = class {
         maxBandwidth: Infinity,
         minChannelsCount: 0,
         maxChannelsCount: Infinity,
+        restrictedVideoCodecs: [],
+        restrictedAudioCodecs: [],
```

Comment:
If it doesn't apply to ABR... it shouldn't be here, that's why I don't like this solution.